gradle.properties

```diff
@@ -1,7 +1,6 @@
-enablePublishing=false
 keikoVersion=3.4.0
-kotlinVersion=1.3.20
+kotlinVersion=1.3.70
```

Comment:
We've been running 1.3.71 in other repos.

Reply:
I'm going to leave it as is for now and figure out a way to include pushing the kork-bom kotlinVersion to the kork consumers as part of autobumping